README.md

```diff
@@ -15,8 +15,28 @@ The gulpfile is 80% from the [gulp-webapp](https://github.com/yeoman/generator-g
 
 There are two commands, both run via npm.
 
-- `npm run build` - clean & build everything and put it into dist folder
-- `npm run serve` - serve the pages and utilize live reload on changes to styles, fonts, images, scripts and HTML.
+- `npm run build` or `gulp build` or `gulp` - clean & build everything and put it into dist folder
```

Comment:
I'm actually in favor of only calling out the usage of `npm run build` and not using gulp directly. The reasoning behind this is that we're then using the local version of gulp which is locked to the project versus the global gulp which could be anything on the user's system.